### ISSUE TITLE
meta.deps does not works

### ISSUE THREAD

**Issue by papasnippy** [systemjs/systemjs#1130]
Part of config:

```
defaultJSExtensions: true,
map: {
	"jquery": "bower_components/jquery/dist/jquery.min",
	"jqueryui": "bower_components/jquery-ui/jquery-ui.min",
	"jquery-event-drag": "bower_components/jquery.event.dragdrop/event.drag/jquery.event.drag",
	"slick-core": "bower_components/slickgrid/slick.core"
},
meta: {
	"jquery":{
		format: "amd"
	},
	"jqueryui": {
		format: "global",
		deps: [
			"jquery"
		]
	},
	"jquery-event-drag": {
		format: "global",
		deps: [
			"jquery"
		]
	},
	"slick-core": {
		format: "global",
		deps: [
			"jquery",
			"jqueryui",
			"jquery-event-drag"
		]
	}
}
```

when i require "slick-core" it does not loads (and even does not try) any of dependencies (also it breaks systemjs-builder). This is happened only after i updated systemjs lib. Older versions (0.19.20) works correcty.

**Comment by guybedford** [comment]
Thanks, fixed in https://github.com/systemjs/systemjs/commit/8c43327942d792b07f64d41dd392c51ec456385d.

**Comment by guybedford** [comment]
Released in 0.19.24.
